js/src/mobx/hardwareStore.js

```diff
@@ -55,6 +58,21 @@ export default class HardwareStore {
     }, HW_SCAN_INTERVAL);
   }
 
+  scanTrezor () {
+    return this._api.parity
+      .lockedHardwareAccountsInfo()
+      .then((paths) => {
+        this.pinMatrixRequest = paths.map((path) => {
```

Comment:
As @jacogr pointed out in one of my requests Mobx may not update the listeners in case the assignment doesn't happen in an action:

```js
@action
setPinMatrixRequest= (req) => {
  this.pinMatrixRequest = req
};

[...]
.then((paths) => {
  this.setPinMatrixRequest(paths.map(...));
});
```